### DISCUSSION TITLE
Password reset within Azure Container Instances

### DISCUSSION THREAD

**Discussion by danielgauci** [datalust/seq-tickets#1352] [Help]
Hi there,

What is the recommended process for resetting the admin password specific to Azure Container Instances (ACI)?

Stores are backed by Postgres and Azure Premium files.

The password was not changed for the admin but somehow I cannot get in :(

Rather high priority if you guys could help out ASAP that would be awesome.

Thank you

**Comment by nblumhardt** [comment]
Hi Daniel, thanks for the note. It might be preferable to figure this out via our support@datalust.co address, we can set up a call if necessary, and you'll reach the whole team there.

Just to clarify, which version of Seq are you running? Also, how old is the instance, did you use the `SEQ_FIRSTRUN_...` variables to set up the admin account, and are there other accounts on the same machine? Finally, can you get access to the Postgres instance from a machine with an interactive command-line and the ability to run Docker containers? Thanks!

**Reply by nblumhardt** [reply]
Hi Daniel - quick check-in; I haven't spotted an email from you yet - are you expecting one to have reached us? Cheers!
